libcalico-go/lib/health/health.go

```diff
@@ -134,6 +138,10 @@ type HealthAggregator struct {
 
 	// HTTP server.  Non-nil when there should be a server running.
 	httpServer *http.Server
+
+	// Track whether we have previously reported live and ready overall.
```

Comment:
`s/previously/ever/` ? (Often think of previous as "the immediately previous".)

Reply:
Good idea, will do.